test/validation/api/crypto/test_vectors.h

```diff
@@ -477,6 +606,244 @@ static crypto_test_reference_t aes_gmac_reference[] = {
 	},
 };
 
+/* Test vectors from SP800-38B / CSRC examples:
+ * https://csrc.nist.gov/CSRC/media/Projects/Cryptographic-Standards-and-Guidelines/documents/examples/AES_CMAC.pdf
```

Comment:
The NIST SP citation is sufficient. Any reader can easily Google it if needed. URLs tend to get stale quickly, and checkpatch flags it as too long.